index.js

```diff
@@ -56,30 +79,32 @@ if (process.stdout.isTTY) {
                 return;
               }
 
-              // Fetching committed git files
-              fetchGitDiff( baseBranch ).then((committedGitFiles = []) => {
-                debug(committedGitFiles);
-                new Listr(resolveMainTask({ tasks, committedGitFiles }), {
-                  exitOnError: false,
-                  concurrent: true,
-                  collapse: false
-                })
-                  .run()
-                  .then(() => {
-                    cache.setSync("linted-hash", commitHash);
-                    debug('Cached Current Commit Hash');
-                    log(success("\nVoila! 🎉  Code is ready to be Shipped.\n"));
+              resolveBaseBranch.then((base) => {
+                // Fetching committed git files
```

Comment:
logging the value of `base` would ease the debugging process

Reply:
I haven't forgotten about this. Just got busy and haven't had a chance to circle back to it and make the update yet.